src/components/pages/HomePage/HowPulsarWorks/HowPulsarWorks.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import s from './HowPulsarWorks.module.css'
+import ScreenTitle from '../ui/ScreenTitle/ScreenTitle';
+
+// SVGO breaks the illustration. To fix it, we import it as is.
+import illustrationDesktop from '!!raw-loader!./img/illustration-desktop.svg';
+import illustrationMobile from '!!raw-loader!./img/illustration-mobile.svg';
+
+import BookkeeperIcon from './img/bookkeeper.svg';
+import BrokersIcon from './img/brokers.svg';
+import ZookeeperIcon from './img/zookeeper.svg';
+import ProducerAndConsumerIcon from './img/producer-and-consumer.svg';
+import Slider from '@site/src/components/ui/Slider/Slider';
+
+const cards: CardProps[] = [
+  {
+    title: 'Producer & Consumer',
+    image: <ProducerAndConsumerIcon />,
+    children: (
+      <p>
+        A Pulsar client contains a consumer and a producer.
+        A producer writes messages on a topic.
+        A consumer reads messages from a topic and acknowledges specific messages or all up to a specific message.
+      </p>
+    )
+  },
+  {
+    title: 'Apache Zookeeper',
+    image: <ZookeeperIcon />,
+    children: (
+      <p>
+        Pulsar and BookKeeper use Apache ZooKeeper to save metadata coordinated between nodes,
+        such as a list of ledgers per topic, segments per ledger, and mapping of topic bundles to a broker.
+        It’s a cluster of highly available and replicated servers (usually 3).
+      </p>
+    )
+  },
+  {
+    title: 'Pulsar Brokers',
+    image: <BrokersIcon />,
+    children: (
+      <p>
+        Topics (i.e., partitions) are divided among Pulsar brokers.
+        A broker receives messages for a topic and appends them to the topic’s active virtual file (a.k.a ledger),
+        hosted on the Bookkeeper cluster. Brokers read messages from the cache (mostly) or BookKeeper and dispatch them to the consumers.
+        Brokers also receive message acknowledgments and persist them to the BookKeeper as well.
+        They are stateless (don’t use/need a disk).
+      </p>
+    )
+  },
+  {
+    title: 'Apache Bookkeeper',
+    image: <BookkeeperIcon />,
+    children: (
+      <p>
+        Apache BookKeeper is a cluster of nodes called bookies.
+        Each virtual file (a.k.a ledger) is divided into consecutive segments, and each segment is kept on 3 bookies by default
+        (replicated by the client - i.e., the broker).
```

Comment:
```suggestion
```

This line seems a bit vague. It won't carry too much information and lay between details to abstractions. I suggest we omit it.

Reply:
That is, a user reading this line generally feels confused - it requires more details to explain why and how the broker as the BookKeeper client replicate data. It's an implementation details and I suggest we just deliver the info that the whole BK application will replicate data. Leave the details to concrete docs.

Reply:
A good method to test out the different is A/B test. Barely arguing won't converge so I'm OK with proceeding the content just AS IS.